distribution/changelog.txt

```diff
@@ -2,6 +2,7 @@
 ------------------------------------------------------------------------
 - Feature: [#9029] Open doors with the tile inspector.
 - Feature: [#11231] Change shortcut window list order to be more intuitive, and split it into logical sections.
+- Feature: [#11306] Replacing path textures do not remove additions on it. It handles the switch from queue line to standard path and vice versa.
```

Comment:
This description is too long. Since it was merged before I got the change to approve it, I will modify it in `develop`.

Reply:
I had pretty much the same gripe, but assumed it would be amended before getting merged